MetaMask/metamask-extension
# Updating storybook docs

Fixes: #12618

Explanation: Updating and moving docs to .storybook/ folder and updating the Button component. This will be useful for the upcoming IA Nav work and pointing contributors to an example of docs standard we want to strive for.

Manual testing steps:  
  -  Go to latest CI build of storybook
  -  Go to Documentation Guidelines and see updated section
  -  Go to "Button" component story and check controls work
  - Go to "Button" component docs page and see updated docs

**Images**

![docs page](https://user-images.githubusercontent.com/8112138/145698821-709f4039-56e2-4bdf-959c-734eb1be9d52.png)

<img width="1440" alt="Screen Shot 2021-12-12 at 4 08 22 PM" src="https://user-images.githubusercontent.com/8112138/145698807-deb24d18-c731-4ce4-b169-c2b894536d56.png">

![btn docs](https://user-images.githubusercontent.com/8112138/145698815-90677ae2-6207-4203-ae82-9ebdaae71bab.png)
